lib/mayaUsd/render/vp2RenderDelegate/render_delegate.cpp

```diff
@@ -385,8 +385,12 @@ class MShaderCache final
             const MHWRender::MShaderManager* shaderMgr
                 = renderer ? renderer->getShaderManager() : nullptr;
             if (TF_VERIFY(shaderMgr)) {
+#if MAYA_API_VERSION >= 20240000
+                shader = shaderMgr->getFragmentShader("UsdPreviewSurface", _structOutputName, true);
```

Comment:
Unsure. Having visual results in a unit test would help compare the results. I suspect you want to
- crank the roughness to 1.0
- enable "use specular workflow"
- keep specular color untouched (default is black)

To make sure you end up with a shading that is as flat as the default shader.